cmd/rhoas/main.go

```diff
@@ -112,5 +113,5 @@ func initConfig(f *factory.Factory) error {
 }
 
 func wrapErrorf(err error, localizer localize.Localizer) error {
-	return fmt.Errorf("Error: %w. %v", err, localizer.MustLocalize("common.log.error.verboseModeHint"))
+	return fmt.Errorf("%v  %w. %v", icon.Error(), err, localizer.MustLocalize("common.log.error.verboseModeHint"))
```

Comment:
```suggestion
	return fmt.Errorf("%v %w. %v", icon.Error(), err, localizer.MustLocalize("common.log.error.verboseModeHint"))
```

There should only be one space here